docs/docs/developer-guides/cosmwasm-cli.md

```diff
@@ -40,6 +58,20 @@ This will take time at first but since it is cached, it will be almost instant a
 ```
 nix run "github:ComposableFi/composable/d2845fc731bc3ee418a17cf528336d50f4b39924#devnet-picasso"
 ```
+
+### Setting up the environment to deploy on a local network of Picasso
+
+Once your node is set up on local from the previous step, open the Polkadot-Js explorer to view activity changes on your local network by heading to the development section with the custom endpoint linked [here](https://polkadot.js.org/apps/?rpc=ws%3A%2F%2F127.0.0.1%3A9988#/explorer). If Polkadot-JS fails to load your local network after running the node, it is possible that there was an error during the build process, resulting in the failure to load it correctly. For development support, feel free to ping us in the dev-chat channel on our [Discord](https://discord.com/invite/composable).
```

Comment:
btw support can be in other page. idea is that if error it is self evident to ask on discrod or do nothing. no need to write it.